src/Manager/Environment/effects/removeEnvironment.ts

```diff
@@ -23,27 +23,42 @@ import {
 } from '../environmentReducer';
 import { removeDir } from './removeDir';
 
-const removeLegacyEnvironment = (dispatch: Dispatch, toolchainDir: string) =>
-    removeDir(dispatch, path.dirname(toolchainDir));
+const removeLegacyEnvironment = (toolchainDir: string) =>
+    removeDir(path.dirname(toolchainDir));
 
-const removeNrfutilEnvironment = (dispatch: Dispatch, version: string) =>
-    Promise.all([
-        removeToolchain(version),
-        removeDir(dispatch, sdkPath(version)),
-    ]);
+const removeNrfutilEnvironment = (version: string) => {
+    removeDir(sdkPath(version));
+    try {
+        removeToolchain(version);
+    } catch (err) {
+        throw new Error(
+            `Failed to remove ${sdkPath(version)}, ${err}. ` +
+                `Please remove the installation for ${version} manually as it is broken.`
+        );
```

Comment:
This is printed if removing the toolchain fails but we are printing the directory of the sdk (which is not where the toolchain is located). Is this really correct? You are more into this than I am, so I assume it will be correct, it just looked strange to me.